dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/StaticLinkageChecker.java

```diff
@@ -245,6 +254,38 @@ JarLinkageReport generateLinkageReport(Path jarPath, SymbolReferenceSet symbolRe
     return Optional.of(LinkageErrorMissingMethod.errorAt(reference));
   }
 
+  /**
+   * Returns an {@code Optional} describing the linkage error for the field reference if the
```

Comment:
This is a weird pattern. Usually the optional is empty on failure, but I'm not sure I have a better alternative to suggest.